test/org/sagebionetworks/bridge/models/PagedResourceListTest.java

```diff
@@ -52,62 +69,26 @@ public void canSerialize() throws Exception {
                 new TypeReference<PagedResourceList<AccountSummary>>() {});
 
         assertEquals(page.getTotal(), serPage.getTotal());
-        assertEquals(page.getOffsetBy(), serPage.getOffsetBy());
-        assertEquals(page.getPageSize(), serPage.getPageSize());
-        assertEquals(page.getFilters().get("emailFilter"), serPage.getFilters().get("emailFilter"));
+        assertEquals(100, serPage.getPageSize());
+        assertEquals((Integer)123, serPage.getOffsetBy());
+        assertEquals(startTime, serPage.getStartTime());
+        assertEquals(endTime, serPage.getEndTime());
+        assertEquals("filterString", serPage.getEmailFilter());
+        
+        Map<String,Object> params = page.getRequestParams();
+        Map<String,Object> serParams = serPage.getRequestParams();
+        assertEquals(params.get("offsetBy"), serParams.get("offsetBy"));
+        assertEquals(params.get("pageSize"), serParams.get("pageSize"));
+        assertEquals(params.get("emailFilter"), serParams.get("emailFilter"));
+        assertEquals(params.get("startTime"), serParams.get("startTime"));
+        assertEquals(params.get("endTime"), serParams.get("endTime"));
         
         assertEquals(page.getItems(), serPage.getItems());
     }
     
-    @Test
+    @Test(expected = NullPointerException.class)
     public void offsetByCanBeNull() throws Exception {
         List<AccountSummary> accounts = Lists.newArrayListWithCapacity(2);
-        PagedResourceList<AccountSummary> page = new PagedResourceList<AccountSummary>(accounts, null, 100, 123)
-                .withFilter("emailFilter", "filterString");
```

Comment:
offsetBy is now only a request parameter (there's no reason to calculate nextPageOffsetBy, though we could). Renamed the test as we're now verifying that total, which is required, will throw an error immediately if not provided.